<launch>
  <!-- Custom move_base configuration with reduced inflation for closer obstacle approach -->
  
  <arg name="custom_param_file" default="$(find tour_guide)/param/move_base_params.yaml"/>
  
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find tour_guide)/param/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find tour_guide)/param/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find tour_guide)/param/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find tour_guide)/param/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find tour_guide)/param/dwa_local_planner_params.yaml" command="load" />
    <rosparam file="$(arg custom_param_file)" command="load" />
    
    <!-- Remap cmd_vel to the mux -->
    <remap from="cmd_vel" to="cmd_vel_mux/input/navi"/>
  </node>
</launch>

